packages/quantic/force-app/examples/main/lwc/exampleQuanticSort/templateWithoutCustomSortOptions.html

```diff
@@ -0,0 +1,35 @@
+<template>
```

Comment:
Maybe rename to "templateDefaultSort" ?

It is a bit hard to read the difference between "templateWithCustomSortOptions" and "templateWithoutCustomSortOptions" :D